app/boards/shields/knob_goblin/knob_goblin.conf

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2022 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+# Uncomment to enable Encoders
+CONFIG_EC11=y
+CONFIG_EC11_TRIGGER_GLOBAL_THREAD=y
+
+# Uncomment the following line to enable the Knob Goblin OLED Display
+CONFIG_ZMK_DISPLAY=y
```

Comment:
These should probably be commented out by default, unless every knob goblin has these

Reply:
Every knob goblin ships with those, but the user may choose to not solder the screen for example. What should I do in this case?

Reply:
In this case, I think the decision should be up to you on whether they should be turned on by default. I would say it seems sane either way.

Reply:
In that case I'd rather leave it like this, that way the default route to install ZMK would work with zero code changes.